implementation/src/main/java/io/smallrye/config/ConfigSourceContext.java

```diff
@@ -8,4 +10,6 @@
 @Experimental("ConfigSource API Enhancements")
 public interface ConfigSourceContext {
     ConfigValue getValue(String name);
+
+    Iterator<String> iterateNames();
```

Comment:
The previous method is about one property but the new method is about a collection of the properties. Since it is contetual info for a config source, what is ConfigValue about?

Reply:
For the contextual property, we just return `ConfigValue` to provide additional info. In fact, internally we changed SR Config to always work with `ConfigValue` so we can have contextual information everywhere.